spotbugs-tests/src/test/java/edu/umd/cs/findbugs/SuppressionMatcherBugReporterTest.java

```diff
@@ -0,0 +1,55 @@
+package edu.umd.cs.findbugs;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class SuppressionMatcherBugReporterTest {
+
+    @Mock
+    private SuppressionMatcher matcher;
+
+    @Mock
+    private BugReporter upstreamReporter;
+
+    @InjectMocks
+    SuppressionMatcherBugReporter reporter;
+
+    @Test
+    void shouldNotCallUpstreamReporterIfBugSuppressed() {
+        // given
+        BugInstance suppressed = mock(BugInstance.class);
```

Comment:
Why mix annotation-based and method-based mocks? This could become another annotated field.

Reply:
There are three tests, one calls the bug instance `suppressed`, the second calls it `reported` and the third one does not create a bug instance, so I think it makes sense that @mafor used a local variable here

Reply:
There is no need for the tests to use different names for the instance. As I see it, we just have 2/3 tests creating an instance mock, so it makes sense to be consistent with other values by making it an annotated field.

Reply:
I have made the change to create the instance with `@Mock`